Formula/mas.rb

```diff
@@ -13,27 +13,18 @@ class Mas < Formula
     sha256 cellar: :any, catalina:      "163eb9cfdfed3d8fbda133b4079d104ad687f1ddb71d70d5661d02b22f562e76"
   end
 
-  depends_on "carthage" => :build
   depends_on :macos
   if Hardware::CPU.arm?
     depends_on xcode: ["12.2", :build]
   else
-    depends_on xcode: ["11.4", :build]
+    depends_on xcode: ["12.0", :build]
   end
 
   def install
-    # Working around build issues in dependencies
-    # - Prevent warnings from causing build failures
-    # - Prevent linker errors by telling all lib builds to use max size install names
-    xcconfig = buildpath/"Overrides.xcconfig"
-    xcconfig.write <<~EOS
-      GCC_TREAT_WARNINGS_AS_ERRORS = NO
-      OTHER_LDFLAGS = -headerpad_max_install_names
-    EOS
-    ENV["XCODE_XCCONFIG_FILE"] = xcconfig
-
-    # Only build necessary dependencies
-    system "carthage", "bootstrap", "--platform", "macOS", "Commandant"
+    # system "script/build"
+    system "script/version"
+    system "swift", "build", "--disable-sandbox", "--manifest-cache", "none", "--configuration", "release",
+      "--arch", "arm64", "--arch", "x86_64"
```

Comment:
Doesn't this try an build a universal binary?

Reply:
It does, as it did before. We have future work to make this architecture-specific.

Reply:
Would passing the right architecture to `--arch` here build something architecture specific? If so, I think we can just make this
```suggestion
    system "swift", "build", "--disable-sandbox", "--manifest-cache", "none", "--configuration", "release",
      "--arch", Hardware::CPU.arch
```

Reply:
Removing the `--arch` flags would build for the current machine's architecture, but would change the path where it is built. We have some additional work before we get there. Changes to our build, install, and packaging scripts are necessary.

Reply:
Where would it get installed? Does it affect `contrib/completion`? If not, then it seems to me that dropping the `--arch` flags would change only one more line here.

Reply:
Can we `inreplace` the path in your install script here, so that it doesn't affect your package script?

Reply:
Potentially, if you think it's worth the hassle. `mas` 1.8.1 was also a universal binary, so this isn't a new change. I'm planning work to simplify and improve this for a future release—possibly as soon as 1.8.3.

Reply:
> Potentially, if you think it's worth the hassle.

Please do. We've been very explicit about not supporting universal binaries, and I'd rather avoid users jumping to the conclusion that because building them for one formula can easily be done, it must be doable for others. (If they ever stumble upon the fact that `mas` is actually a universal binary on their system.)

Reply:
Will do. I'd hoped we could get our fixes out before doing this work, but it sounds like we'll need to get it done now.

Reply:
https://github.com/mas-cli/mas/pull/365#commits-pushed-3e88d28